test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java

```diff
@@ -112,177 +104,12 @@ private static void init()
         // ... and start resizing
         robot.mousePress( InputEvent.BUTTON1_MASK );
         robot.mouseMove(bounds.x + bounds.width + 20, bounds.y + bounds.height + 15);
-        Util.waitForIdle(robot);
+        robot.mouseRelease(InputEvent.BUTTON1_MASK);
```

Comment:
This is the actual change, mouseRelease is moved from the end of the test to the line above.